lib/api/Cargo.toml

```diff
@@ -117,7 +118,7 @@ jit = ["engine"]
 js = ["wasm-bindgen", "js-sys"]
 js-default = ["js", "std", "wasm-types-polyfill"]
 
-wasm-types-polyfill = ["js", "wasmparser"]
+wasm-types-polyfill = ["js", "wasmparser", "js-serializable-module" ]
```

Comment:
Why we require the module to be serializable when we do the polyfill? It shouldn't be required